workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -140,9 +82,19 @@ public static void DomainUnloadShutdown()
             ScriptBehaviourUpdateOrder.UpdatePlayerLoop();
         }
 
-        private void LoadLevel(WorkerBase worker)
+        private void CreateWorker(ConnectionConfig config, Vector3 origin)
         {
-            Instantiate(Level, worker.Origin, Quaternion.identity);
+            if (config.WorkerType.Equals("UnityGameLogic"))
+            {
+                PlayerLifecycleConfig.CreatePlayerEntityTemplate = PlayerTemplate.CreatePlayerEntityTemplate;
```

Comment:
Why does this need to be conditional? If its set on the client, does it matter? 

Related - does this even belong here?

Reply:
it was only set for the gamelogic worker in the previous code, simply keeping the same logic here. 
The overall idea would be to factor it out into it's own monobehavior/gameobject, but I didn't want to make the PR even bigger